{{cookiecutter.plugin_repo}}/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
```

Comment:
let's drop .travis from the cookiecutter, it's too specific for each of the plugins.